pmm-app/src/pmm-check/CheckPanel.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Table } from 'antd';
+import { mount, ReactWrapper } from 'enzyme';
+import { CheckPanel, CheckPanelProps, CheckPanelState } from './CheckPanel';
+import { activeCheckStub } from './__mocks__/stubs';
+
+jest.mock('../react-plugins-deps/components/helpers/notification-manager');
+
+jest.mock('./Check.service');
+
+describe('CheckPanel::', () => {
+  CheckPanel.prototype.componentDidMount = jest.fn();
+
+  it('should accept a title parameter and display it as a panel title', async () => {
+    const props: any = {
+      width: 1200,
+      height: 450,
+      options: {
+        title: 'DB CHECKS',
+      },
+    };
+
+    const root: ReactWrapper<CheckPanelProps, CheckPanelState, CheckPanel> = mount(<CheckPanel {...props} />);
+
+    // Check for the panel title passed as component prop
+    expect(
+      root
+        .find('[data-testid="db-check-panel"] > div > div > div')
+        .at(0)
+        .text()
+    ).toEqual('DB CHECKS');
+
+    await root.instance().fetchAlerts();
+    root.update();
+
+    expect(root.state('dataSource')).toEqual(activeCheckStub);
+    expect(root.state().loading).toEqual(false);
+
+    const table = root.find('[data-testid="db-check-panel"]').find(Table);
```

Comment:
Please replace data-testid with data-qa attribute, we're using this one now

Reply:
This is super custom, why don't we adopt the established naming practices?
For example: https://testing-library.com/docs/dom-testing-library/api-queries, there are many others.

Reply:
Hm, I don't see any 'data-qa' attributes in the PMM-2.0 branch.